### ISSUE TITLE
Selecting an article from the JAWS 2019 List of Articles does not work

### ISSUE THREAD

**Issue by bethwhitmer** [FreedomScientific/standards-support#202]
### Summary

When I select an article from the JAWS 2019 List of Articles, I am not being navigated to the article. JAWS instead announces "main region" and starts reading from there. 

Example:
1. Go to http://terrillthompson.com/tests/article.html (verified on another page as well) in IE11, Firefox, or Chrome. 
2. Press Insert+F3 to bring up the JAWS Lists and then select the Articles list.
3. Select the Example 2 article from the list and press Enter to navigate to it. JAWS does not navigate to the article and instead announces "main region" and then starts reading from the start of the main region. 

### Expected result

Just like all other JAWS lists, I expected that I would be navigated to the element that I selected from the list (in this case the Example 2 article). 

### Actual result

JAWS started reading from the top of the main region.

### Example

http://terrillthompson.com/tests/article.html or any page that contains article elements.

#### JAWS version and build number

 JAWS 2019.1904.60

#### Operating System and version

Windows 7

#### Browser and version:

 IE11, Firefox, Chrome

**Comment by lamador** [comment]
This issue was reproducible using the latest version of JAWS (2020.2006.12), Chrome (83.0.4103.97 (Official Build) (64-bit)), Firefox (77.0.1 (64-bit)), and IE11 (11.836.18362.0).  For claification, there are two `article` elements identified on the example page and neither can be navigated to via the articles list window.

**Comment by stevefaulkner** [comment]
works in JAWS 2024.2310.70
